zeithistorische-forschungen.csl

```diff
@@ -0,0 +1,393 @@
+      </if>
+    </choose>
+  </macro>
+  <macro name="published">
+    <choose>
+      <if type="article-newspaper" match="none">
+        <text variable="publisher-place"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="orig-date">
+    <group delimiter=" ">
+      <choose>
+        <if is-uncertain-date="issued">
+          <text term="circa" form="short"/>
+        </if>
+      </choose>
+      <date variable="original-date" form="numeric"/>
+    </group>
+  </macro>
+  <macro name="date">
+    <choose>
+      <if variable="issued">
+        <date date-parts="year" form="numeric" variable="issued"/>
+      </if>
+      <else>
+        <text term="no date"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="issued">
+    <group prefix=", " delimiter=", ">
+      <group delimiter=" ">
+        <text variable="genre"/>
+        <choose>
+          <if type="report">
+            <text variable="number"/>
+          </if>
+        </choose>
+      </group>
+      <text variable="event"/>
+      <text macro="collection"/>
+      <text macro="edition"/>
+    </group>
+    <choose>
+      <if type="book chapter thesis" match="any">
+        <group prefix=", " delimiter=" ">
+          <text macro="published"/>
+          <text macro="date"/>
+        </group>
+      </if>
+      <else-if type="article-journal article-magazine" match="any">
+        <group prefix=" " delimiter=" ">
+          <group delimiter=", ">
+            <text macro="volume"/>
+            <text macro="secondary-container-contributors"/>
+          </group>
+          <text prefix="(" macro="date" suffix=")"/>
+        </group>
+      </else-if>
+      <else>
+        <group prefix=", " delimiter=", ">
+          <text macro="published"/>
+          <text macro="date"/>
+        </group>
+      </else>
+    </choose>
+    <choose>
+      <if type="manuscript">
+        <text prefix=" (" value="Typoskript" suffix=")"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="pages">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper chapter" match="any">
+        <group delimiter=". ">
+          <label variable="page" form="short"/>
+          <text variable="page"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="access">
+    <group delimiter=", ">
+      <choose>
+        <if type="webpage" match="any">
+          <date variable="accessed" form="numeric"/>
+        </if>
+      </choose>
+      <text variable="URL" prefix="URL: &lt;" suffix="&gt;"/>
+    </group>
+  </macro>
+  <macro name="citation-locator">
+    <group delimiter=". ">
+      <label variable="locator" form="short"/>
+      <text variable="locator"/>
+    </group>
+  </macro>
+  <citation et-al-min="4" et-al-use-first="1" disambiguate-add-names="true">
+    <layout suffix="." delimiter="; ">
+      <choose>
+        <if position="ibid-with-locator">
+          <text term="ibid" text-case="capitalize-first" suffix="."/>
```

Comment:
Ist ebd. gewollt? In den Zitierregeln sehe ich dazu nichts.

Reply:
Ganz herzlichen Dank für das ausführliche Review! Ebd./ibid. ist nicht spezifiziert, wird jedoch gebraucht. Siehe https://zeithistorische-forschungen.de/1-2021/5909 und https://zeithistorische-forschungen.de/1-2021/5908